packages/owl-bot/src/logger.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 Google LLC
```

Comment:
To appease the header linter:

```suggestion
// Copyright 2022 Google LLC
```

Reply:
Well, this file is not auto generated. Changing copyring header seems wrong. I'd fix the bot, but is it difficult?